.buildkite/pipeline.yml

```diff
@@ -141,8 +141,16 @@ steps:
   - group: "Plane Examples"
     steps:
       - label: ":computer: Agnesi linear hydrostatic mountain experiment"
-        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config plane --x_max 240000.0 --z_max 25000.0 --x_elem 80 --z_elem 90 --dt 1.5secs --t_end 14400.0secs --z_stretch false --vert_diff false --kappa_4 1e6 --dt_save_to_disk 3600secs --initial_condition AgnesiHProfile --discrete_hydrostatic_balance true --topography Agnesi --rayleigh_sponge true --alpha_rayleigh_uh 0.0 --alpha_rayleigh_w 0.1 --job_id plane_agnesi_mountain_test"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config plane --x_max 240000.0 --z_max 25000.0 --x_elem 80 --z_elem 90 --dt 1.5secs --t_end 14400.0secs --z_stretch false --vert_diff false --kappa_4 1e6 --dt_save_to_disk 3600secs --initial_condition AgnesiHProfile --topography Agnesi --rayleigh_sponge true --alpha_rayleigh_uh 0.0 --alpha_rayleigh_w 0.1 --job_id plane_agnesi_mountain_test --use_reference_state false"
         artifact_paths: "plane_agnesi_mountain_test/*"
+      
+      - label: ":computer: Schar mountain experiment (uniform grid)"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --config plane --x_max 120000.0 --z_max 25000.0 --x_elem 100 --z_elem 45 --dt 0.4secs --t_end 14400.0secs --z_stretch false --vert_diff false --kappa_4 5e7 --dt_save_to_disk 3600secs --initial_condition ScharProfile --topography Schar --rayleigh_sponge true --alpha_rayleigh_uh 0.0 --alpha_rayleigh_w 0.1 --job_id plane_schar_mountain_test_uniform --use_reference_state false"
```

Comment:
Any reason why z_max is different from the stretched case?

Reply:
Simplest way to generate the monotonically increasing grid sizes following what we do on the sphere with 45 levels; I can otherwise also maintain the domain heights and select the dz_bottom, dz_top arguments within ` Meshes.GeneralizedExponentialStretching(dz_bottom, dz_top)` to enforce a specific stretching profile.